lib/bot.c

```diff
@@ -0,0 +1,422 @@
+#include <time.h>
+#include "bot.h"
+#include "board.c"
+
+/**
+ * Declare global variable for set winner condition.
+ */
+int winCondition;
```

Comment:
var winCondition ini bisa diakses sama modul-modul di file game.c gak?

Reply:
bisa bang, itu di declare diluar function/procedure

here's example, coba manggil `winCondition` di main module
![image](https://github.com/muhammadfajri-tif/tubes-03-1b/assets/149763217/e940bcc1-e310-4045-8028-cf8e60626c50)

here's the output:
![image](https://github.com/muhammadfajri-tif/tubes-03-1b/assets/149763217/28f9e79d-3145-4eee-b19c-c8ab200ece1c)